### ISSUE TITLE
Conker: Live & Reloaded [4D530051]

### ISSUE THREAD

**Issue by furrysnack** [Cxbx-Reloaded/game-compatibility#154]
### Description

The game is an LTCG title, and isn't supported by HLE. Starting it with LLE enabled leaves it on a black screen.

---

### Game info

[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1231929/Xbe.txt)

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1231930/KrnlDebug.txt)

---

### Status
Black Screen

---

### Screenshots

Image 1: Emulation with HLE

![image](https://user-images.githubusercontent.com/31105997/29422474-52859a80-8346-11e7-8fc7-dc9fdcd29f65.png)

Image 2: Emulation with LLE

![image](https://user-images.githubusercontent.com/31105997/29422493-63dc2a92-8346-11e7-8005-65623ad1593b.png)

---

### Additional Information

The .xbe file I used is from the PAL region.

**Comment by Voxel9** [comment]
Conker unfortunately hangs indefinitely on a black screen in both HLE and LLE...

---
[KrnlDebug-HLE.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1877235/KrnlDebug-HLE.txt)
[KrnlDebug-LLE.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1877234/KrnlDebug-LLE.txt)

**Comment by ergo720** [comment]
It's because a call to NtQueryVirtualMemory succeeds on cxbx but the game is expecting it to fail instead and so it will keep on trying, hence the hang. If you run a debug version you should see the calls in the log. @LukeUsher told me that it actually renders if you hack the function and make it fail instead.

**Comment by Voxel9** [comment]
Now the game displays a test-case message, followed by an exception:

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1977535/KrnlDebug.txt)
---
```
---------------------------
Cxbx-Reloaded
---------------------------
Please report that Conker: Live and Reloaded shows this test-case: Validate this function!
In EmuPatch_D3DDevice_EndVisibilityTest_0 (c:\projects\cxbx-reloaded\src\cxbxkrnl\emud3d8.cpp line 2684)
---------------------------
OK   
---------------------------
```
```
---------------------------
Cxbx-Reloaded
---------------------------
Received Exception Code 0xC0000005 @ EIP := 0x00538A21(=D3DDevice_SetPixelShaderConstant_4_UNPATCHED+0x181)

  Press "OK" to terminate emulation.
  Press "Cancel" to debug.
---------------------------
OK   Cancel   
---------------------------
```

**Comment by Voxel9** [comment]
Conker now gets up to the boot splash with HLE, before freezing. 

![image](https://user-images.githubusercontent.com/16278868/40626610-9a78e70c-62b1-11e8-808f-6ed2b7f80e11.png)

**Comment by APGRoboCop** [comment]
I'm still getting the Black Screen when I load up that game from the PAL Region also from Sept 11th 2018 build. Even removing that bootintro.xmv didn't work neither.

**Comment by KetchupWithFries** [comment]
Yeah, sadly can't get it to work yet >.< Hopefully, we won't have to wait too long for it to start working.

**Comment by C0NN0RDAM** [comment]
Any news yet about this game? I remember when I played as a child! I want to play it so badly again :(

**Comment by trveg** [comment]
It's my dream to play this game on PC

**Comment by eldexterr** [comment]
Please!

**Comment by franchise2099** [comment]
How did every other emulator gain so much traction? Rpcs3 is moving at am incredible rate for how crazy complex that one is.  I've heard a bunch of people say Xbox emulation was overlooked because most games were available on PC. there are still some gems within the original Xbox library.  Really want to play this game on my PC.

**Comment by martinvolantin** [comment]
I get this output in the kernel log on version 368428b on HLE
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3315649/KrnlDebug.txt)

**Comment by Bx2man** [comment]
![Untitled (1075x498)](https://user-images.githubusercontent.com/13337034/74791156-b3deb100-5287-11ea-9308-834de02e04bb.jpg)

Plays intro video, then displays this output. There is also sound, but it jumps around randomly playing different sounds for a while before it crashes. 
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4222549/KrnlDebug.txt)

**Comment by bryce13950** [comment]
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4777123/KrnlDebug.txt)
It now seems to be landing on the loading screen, and freezing at that point.

**Comment by bryce13950** [comment]
The title id for this is MS-081 v1.02

**Comment by mnadareski** [comment]
On build df22c8b:
- Attempted to boot and got locked on the initial title screen
![image](https://user-images.githubusercontent.com/4381583/119303709-0fca7f00-bc1b-11eb-801f-9ffa66bf9603.png)

The following messages were in the log:
```
[0x49CC] INFO : VTXSH   Please report that Conker: Live and Reloaded shows the following message:
LOG_TEST_CASE: g_Xbox_VertexShader_Handle != Handle
In CxbxImpl_SetVertexShader (D:\a\Cxbx-Reloaded\Cxbx-Reloaded\src\core\hle\D3D8\XbVertexShader.cpp line 1512)
[0x49CC] INFO : VTXSH   Please report that Conker: Live and Reloaded shows the following message:
LOG_TEST_CASE: Not in shader program mode after SetVertexShader trampoline
In CxbxImpl_SetVertexShader (D:\a\Cxbx-Reloaded\Cxbx-
[0x1894] INFO : PSHB    Please report that Conker: Live and Reloaded shows the following message:
LOG_TEST_CASE: HLE_draw_inline_buffer
In HLE_draw_inline_buffer (D:\a\Cxbx-Reloaded\Cxbx-Reloaded\src\core\hle\D3D8\XbPushBuffer.cpp line 139)
```

Going along with the above hidden comment, this appears to be v1.02 not v1.03. If there is a v1.03, I'll open a new issue for this version instead.

**Comment by Margen67** [comment]
> 
> 
> On build df22c8b:
> 
>     * Attempted to boot and got locked on the initial title screen
>       ![image](https://user-images.githubusercontent.com/4381583/119303709-0fca7f00-bc1b-11eb-801f-9ffa66bf9603.png)
> 
> 
> The following messages were in the log:
> 
> ```
> [0x49CC] INFO : VTXSH   Please report that Conker: Live and Reloaded shows the following message:
> LOG_TEST_CASE: g_Xbox_VertexShader_Handle != Handle
> In CxbxImpl_SetVertexShader (D:\a\Cxbx-Reloaded\Cxbx-Reloaded\src\core\hle\D3D8\XbVertexShader.cpp line 1512)
> [0x49CC] INFO : VTXSH   Please report that Conker: Live and Reloaded shows the following message:
> LOG_TEST_CASE: Not in shader program mode after SetVertexShader trampoline
> In CxbxImpl_SetVertexShader (D:\a\Cxbx-Reloaded\Cxbx-
> [0x1894] INFO : PSHB    Please report that Conker: Live and Reloaded shows the following message:
> LOG_TEST_CASE: HLE_draw_inline_buffer
> In HLE_draw_inline_buffer (D:\a\Cxbx-Reloaded\Cxbx-Reloaded\src\core\hle\D3D8\XbPushBuffer.cpp line 139)
> ```
> 
> Going along with the above hidden comment, this appears to be v1.02 not v1.03. If there is a v1.03, I'll open a new issue for this version instead.

1.03 is in the original post.

**Comment by mnadareski** [comment]
Okay, I'll migrate my comments to a new item, then.
